samples/subscriptions.js

```diff
@@ -661,13 +664,13 @@ const cli = require(`yargs`)
   .example(`node $0 list`)
   .example(`node $0 list my-topic`)
   .example(`node $0 create my-topic worker-1`)
-  .example(`node $0 create-flow my-topic worker-1 -m 5`)
   .example(`node $0 create-push my-topic worker-1`)
   .example(`node $0 modify-config my-topic worker-1`)
   .example(`node $0 get worker-1`)
   .example(`node $0 listen-messages my-subscription`)
   .example(`node $0 sync-pull my-project my-subscription`)
   .example(`node $0 listen-errors my-subscription`)
+  .example(`node $0 listen-flow-control -m 5`)
```

Comment:
I think we need to specify a subscription name here as the first param, yes?

Reply:
That's right.